.sauce/testcafe.yml

```diff
@@ -1,18 +1,37 @@
 apiVersion: v1alpha
-metadata:
-  name: Testing TestCafe Support
-  tags:
-    - e2e
-    - release team
-    - other tag
-  build: Release $CI_COMMIT_SHORT_SHA
-files:
-  - ./tests/e2e/testcafe/example.test.js
-suites:
-  - name: "saucy test"
-    match: ".*.(spec|test).[jt]s$"
-image:
-  base: saucelabs/stt-testcafe-node
-  version: latest
+kind: testcafe
 sauce:
   region: us-west-1
+  concurrency: 1 # Controls how many suites are executed at the same time (sauce test env only).
+  metadata:
+    name: Testing Testcafe Support
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+suites:
+  - name: "saucy test"
+    browserName: "firefox" # Although Testcafe supports triggering one test in multiple browsers, it'd better here to split them into every suite to indicate each suite has its own test point.
```

Comment:
`browserName` or `browser` ?

Reply:
@dpgraham discussed with @alexplischke , it should be `browserName`

Reply:
I had suggested `browserName` since neither `browser` nor `browserName` is native to testcafe.

Arguments for`browserName`:
Because it's closer to what we use internally.

Arguments for`browser`:
Because it's closer to the native testcafe attribute `browsers`.

Happy to go with either :)